src/benchmarklib/tpch/tpch_queries.hpp

```diff
@@ -7,7 +7,7 @@ namespace opossum {
 constexpr size_t NUM_TPCH_QUERIES = 22;
 constexpr size_t NUM_SUPPORTED_TPCH_QUERIES = 7;
 
-extern const char* tpch_queries[21];
+extern const char* tpch_queries[NUM_TPCH_QUERIES];
 extern const char* tpch_query_templates[NUM_TPCH_QUERIES];
```

Comment:
This is not being used anywhere. Is there some reason it is still in there?

Reply:
Insert Gandalf "I have no memory of this place" meme. No.